src/js-executor/index.js

```diff
@@ -40,7 +40,7 @@ class JsExecutor {
 
   async execute(jsCode, jsLibs, theme) {
     const loadedScripts = (this.iframe.contentDocument || this.iframe.document).getElementsByTagName('script').length;
-    // 2 scripts by default: INIT_SCRIPT + JQuery
+    // 2 scripts by default: INIT_SCRIPT + kotlin stdlib
```

Comment:
Hi! Why `stdlib`? We loaded 1 external lib - JQuery.

Reply:
In the `reloadIframeScripts` two scripts are hard coded. One is the `kotlin.js` file and the other is the INIT_SCRIPT. jQuery on the other hand is added in `getJsLibraries` and therefore part of the `jsLibs`.

I guess things changed at some point in time and the comment was not updated.

Reply:
Yeah. Sure, It's my fault